app/utils/purecloud-environment.js

```diff
@@ -1,8 +1,8 @@
-const ENV_REG_EXP = /(inin[dts]ca|mypurecloud.*|localhost).*/i;
+const ENV_REG_EXP = /(inin[dts]ca|[a-zA-Z]*\.mypurecloud\..*|localhost).*/i;
```

Comment:
`.*` is unnecessary as it's "match any amount of anything except newline" and not in the capture group.

`[dts]` should be changed to `[dt]` since we don't have an SCA environment anymore.